MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,162 @@
 import UIKit
 
-final class MovieQuizViewController: UIViewController {
+final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
+
+    // MARK: - QuestionFactoryDelegate
+    
+    func didRecieveNextQuestion(question: QuizQuestion?) {
```

Comment:
Можно включить [проверку правописания](https://sarunw.com/posts/spell-checking-in-xcode/), чтобы не пропускать опечатки
```suggestion
    func didReceiveNextQuestion(question: QuizQuestion?) {
```